week3/body_mass.py

```diff
@@ -11,21 +11,30 @@
 
 def bmi_calculator(weight, height):
 
-    BMI = weight / height**2
+    bmi = weight / height**2
 
-    TABLE = (
-        ((29.9), "Obese"),
-        ((24.9), "Overweight"),
-        ((18.5), "Normal"),
-        ((0), "Underweight"),
-        )
+    OVERWEIGHT, NORMAL, UNDERWEIGHT = 29.9, 24.9, 18.5
 
-    return [ TABLE[TABLE.index(data)][1] for data in TABLE if BMI > data[0] ][0]
+    if bmi > OVERWEIGHT:
+
+        return "Obese"
+
+    elif OVERWEIGHT >= bmi > NORMAL:
+
+        return "Overweight"
+
+    elif NORMAL >= bmi > UNDERWEIGHT:
+
+        return "Normal"
+
+    elif bmi <= UNDERWEIGHT:
+
+        return "Underweight"
 
```

Comment:
yeah, much better!
also (sorry for insisting on the syntax but it is very important for the future), avoid adding unnecessary new lines.
From the pep8: "Use blank lines in functions, sparingly, to indicate logical sections."
See: https://www.python.org/dev/peps/pep-0008/#blank-lines